cairo/tests/ethereum/cancun/test_trie.py

```diff
@@ -149,56 +139,63 @@ def test_get_branches(self, cairo_run, obj, level):
     @pytest.mark.slow
     @settings(max_examples=5)  # for max_examples=2, it takes 239.03s in local
     @given(obj=st.dictionaries(nibble, bytes32))
-    def test_patricialize(self, cairo_run, obj: Mapping[Bytes, Bytes]):
-        assert patricialize(obj, Uint(0)) == cairo_run("patricialize", obj, Uint(0))
+    def test_patricialize(self, cairo_run_py, obj: Mapping[Bytes, Bytes]):
+        assert patricialize(obj, Uint(0)) == cairo_run_py("patricialize", obj, Uint(0))
 
 
 class TestTrieOperations:
-    @given(trie=..., key=...)
-    def test_trie_get_TrieAddressAccount(
-        self, cairo_run, trie: Trie[Address, Optional[Account]], key: Address
-    ):
-        [trie_cairo, result_cairo] = cairo_run("trie_get_TrieAddressAccount", trie, key)
-        result_py = trie_get(trie, key)
-        assert result_cairo == result_py
-        assert trie_cairo == trie
-
-    @given(trie=..., key=...)
-    def test_trie_get_TrieBytes32U256(
-        self, cairo_run, trie: Trie[Bytes32, U256], key: Bytes32
-    ):
-        [trie_cairo, result_cairo] = cairo_run("trie_get_TrieBytes32U256", trie, key)
-        result_py = trie_get(trie, key)
-        assert result_cairo == result_py
-        assert trie_cairo == trie
-
-    @given(trie=..., key=..., value=...)
-    def test_trie_set_TrieAddressAccount(
-        self, cairo_run, trie: Trie[Address, Account], key: Address, value: Account
-    ):
-        cairo_trie = cairo_run("trie_set_TrieAddressAccount", trie, key, value)
-        trie_set(trie, key, value)
-        assert cairo_trie == trie
-
-    @given(trie=..., key=..., value=...)
-    def test_trie_set_TrieBytes32U256(
-        self, cairo_run, trie: Trie[Bytes32, U256], key: Bytes32, value: U256
-    ):
-        cairo_trie = cairo_run("trie_set_TrieBytes32U256", trie, key, value)
-        trie_set(trie, key, value)
-        assert cairo_trie == trie
+    class TestGet:
+        @given(trie=..., key=...)
+        def test_trie_get_TrieAddressAccount(
+            self, cairo_run, trie: Trie[Address, Optional[Account]], key: Address
+        ):
+            [trie_cairo, result_cairo] = cairo_run(
```

Comment:
```suggestion
            trie_cairo, result_cairo = cairo_run(
```

I guess that you don't need the brackets here and in the following test cases